File: 162-find-peak-element/find-peak-element.cpp
class Solution {
public:
    int findPeakElement(vector<int>& nums) {
         if (nums.size() == 0) {
        return -1; 
    }

    if (nums.size() == 1) {
        return 0; 
    }

    for (int i = 0; i < nums.size(); i++) {
      
        if ((i == 0 || nums[i - 1] < nums[i]) && (i == nums.size() - 1 || nums[i + 1] < nums[i])) {
            return i;
        }
    }

    return -1;
    }
};

File: 162-find-peak-element/find-peak-element.py
class Solution:
    def findPeakElement(self, nums: List[int]) -> int:
        for  i in range(0,len(nums)):
            if len(nums)==0:
                return 0
            if len(nums)-1 == i:
                return i 

            if nums[i-1] < nums[i] and nums[i+1] < nums[i]:
                return i
        return 0